sys/usb/usbus/dfu/Kconfig

```diff
@@ -0,0 +1,34 @@
+
+# Copyright (c) 2020 Mesotic SAS
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_USEMODULE_USBUS_DFU
+    bool "Configure USBUS DFU"
+    depends on USEMODULE_USBUS_DFU
+    help
+        Configure the USBUS DFU module via Kconfig.
+
+if KCONFIG_USEMODULE_USBUS_DFU
+
+config USB_DFU_DETACH_TIMEOUT_MS
+    unsigned short int
```

Comment:
There is only `int` type. The restrictions on the possible values can be added by `range <min> <max>`.

Reply:
Fixed.